frontend/src/components/phones/onboarding/RelayNumberConfirmationModal.tsx

```diff
@@ -0,0 +1,91 @@
+import { FormEventHandler, ReactElement, ReactNode, useRef } from "react";
+import { useLocalization } from "@fluent/react";
+import {
+  OverlayContainer,
+  FocusScope,
+  useDialog,
+  useModal,
+  useOverlay,
+  usePreventScroll,
+} from "react-aria";
+import { OverlayProps } from "@react-aria/overlays";
+import styles from "./RelayNumberConfirmationModal.module.scss";
+import { CloseIcon, InfoIcon } from "../../Icons";
+import { Button } from "../../Button";
+import { formatPhone } from "../../../functions/formatPhone";
+
+export type Props = {
+  onClose: () => void;
+  confirm: () => void;
+  relayNumber: string;
+};
+
+export const RelayNumberConfirmationModal = (props: Props) => {
+  const { l10n } = useLocalization();
+
+  const onSubmit: FormEventHandler = (event) => {
+    event.preventDefault();
+  };
+
+  return (
+    <OverlayContainer>
+      <PickerDialog
+        title={`test`}
+        onClose={() => props.onClose()}
+        isDismissable={true}
+      >
+        <button
+          className={styles["cancel-button"]}
+          onClick={() => props.onClose()}
+        >
+          <CloseIcon alt="Close" />
+        </button>
+        <form onSubmit={onSubmit}>
```

Comment:
If the `<Button>` does not submit the form (i.e. there's an `onClick` handler on it, rather than it being `type="submit"` and the form's `onSubmit` doing the handling), then the `<form>` probably isn't needed :)

Reply:
😭 love how I just left the preventDefault in there, like that was doing anything important, LOL. Thanks for the cleanup!